config/dispatcher/dispatcher.ex

```diff
@@ -50,6 +50,23 @@ defmodule Dispatcher do
     Proxy.forward conn, path, "http://resource/jobs/"
   end
 
+  ###############################################################
+  # reports
+  ###############################################################
+
+  post "/reports", %{ accept: [:json], layer: :api } do
+    allowed_groups = Plug.Conn.get_req_header(conn, "mu-auth-allowed-groups")
+    if Enum.any?(allowed_groups, fn group -> group =~ ~r/"name":"admin"/ end) do
```

Comment:
does loket do it like that too? This is bypassing mu auth completely.

Reply:
No this was really my addition. This is for manually regenerating a report, so it doesn't pass the authorization service. I added this "shield" so the service can't be "overflooded". But it is quite a messy solution and you could argue that this wouldn't even be necessary. Viewing the reports is shielded by the authorization as usual, and that's of course most important.
Shall I get rid of this extra check?

Reply:
I'd remove it or create a "dispatcher-dev.ex" file and override the config in the docker-compose.dev.yml if it's a test endpoint :) . If the endpoint is needed and the shield is an extra safety on top of mu auth, and shouldn't affect the rest, adding a comment explaining it would be nice, so that other members have the context